Dockerfile

```diff
@@ -17,16 +17,17 @@ WORKDIR /mnt/workdir
 # from the mounted directory. But that's not available during the
 # docker build, so we need to copy them in to pre-install the Gems
 
-COPY Gemfile ./Gemfile
-COPY Gemfile.lock ./Gemfile.lock
+COPY Gemfile Gemfile.lock ./
 
 # Gems will be installed under GEM_HOME which is set by the ruby image.
 # See https://hub.docker.com/_/ruby for details.
 
-RUN gem update --system && bundle install && gem cleanup
+RUN gem update --system \
+  && bundle install \
+  && gem cleanup
```

Comment:
I don't know how Dockerfiles are parsed, but in bash, the backslash is redundant if the line ends with `&&`, as in:

```
RUN gem update --system &&
  bundle install &&
  gem cleanup
```

This is even easier to read, but I don't know if it works the same in Dockerfiles.

Reply:
I tried this out and it did not work so I guess docker requires it.